client/bdf-font.h

```diff
@@ -66,6 +66,9 @@ int DrawText(FlaschenTaschen *c, const Font &font, int x, int y,
              const Color &color, const Color *background_color,
              const char *utf8_text);
 
+int RotDrawText(FlaschenTaschen *c, const Font &font, int y, int x,
```

Comment:
I'd call it VerticalDrawText(); 'Rotation' would be another expectation.

This function needs a description in the header.

x and y are swapped which is very confusing.